webpack.prod.config.js

```diff
@@ -0,0 +1,27 @@
+const baseConfig = require('./webpack.config.js');
+const webpack = require('webpack');
+const config = require('config');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const GLOBALS = {
+  API_BASE_URL: JSON.stringify(
+    process.env.API_BASE_URL ||
+      `${config.api.host}:${config.api.port}${config.api.baseUrl}`
+  )
+};
+
+baseConfig.plugins = [
+  new webpack.DefinePlugin(GLOBALS), //define variables to use on development
+  new webpack.DefinePlugin({
+    'process.env': {
+      NODE_ENV: JSON.stringify('production'),
+      BABEL_ENV: JSON.stringify('production')
+    }
+  }),
+  new HtmlWebpackPlugin({
+    title: 'Easter Seals',
+    template: 'index.tmpl.html'
+  })
```

Comment:
Is this plugin, and this configuration required?

Reply:
I think I can answer this one. As far as I know Webpack creates for you a "dynamic index.html" but if you want to fully customize `HtmlWebpackPlugin` bundles any custom file you provide.

Reply:
@matiasbeckerle correct, I needed to be able to control the content of the index file (ie title, viewport, etc)